src/minter-suite-mapping.ts

```diff
@@ -759,11 +723,24 @@ export function handleSetValueProjectConfig<T>(event: T): void {
     event.block.timestamp
   );
   if (minterProjectAndConfig) {
-    handleSetValueGeneric(
-      event,
-      minterProjectAndConfig.projectMinterConfiguration,
-      minterProjectAndConfig.project
+    const config = minterProjectAndConfig.projectMinterConfiguration;
+    if (
+      !(
+        config instanceof Minter || config instanceof ProjectMinterConfiguration
+      )
+    ) {
+      return;
+    }
+
+    handleSetMinterDetailsGeneric(
+      event.params._key.toString(),
+      event.params._value,
+      config
     );
+
+    if (minterProjectAndConfig.project) {
+      minterProjectAndConfig.project.updatedAt = event.block.timestamp;
+    }
```

Comment:
This isn't doing anything since we don't save the project after this. However, I don't think we want to update the updatedAt on the project entity in this case - we just want the handleSetMinterDetailsGeneric to update the ProjectMinterConfiguration's updatedAt timestamp. So I think this can be removed?

Reply:
Currently no updatedAt on the ProjectMinterConfiguration entity so we do actually want to set the updatedAt on the project here